request_handler.py

```diff
@@ -0,0 +1,75 @@
+import importlib
+import methods
+import copy
+
+def execute(request):
+    """
+    Executing requests using the methods defined in the methods module.
+    
+    Request dict format:
+    {"func" : <function name that is in the methods module>,
+     "requests" : <list of dicts, functions and arguments to run in order. Not used if "func" is not equal to "tether">, 
+     "args" : <ordered arguments for func, including data input. For tether, these are the initial input arguments>,
+     "kwargs" : <key worded arguments for func>
+    }
+    
+    input: request (dict)
+    output: <tether function> OR method.func(*args, **kwargs)
+    """
+    
+    args = []
+    kwargs = {}
+    
+    if "args" in request:
+        if isinstance(request["args"], type(None)):
+            pass
+        else:    
+            args = request["args"]
+    
+    if "kwargs" in request:
+        if isinstance(request["kwargs"], type(None)):
+            pass
+        else:    
+            kwargs = request["kwargs"]
+        
+    func_name = request["func"]
```

Comment:
This is an interesting idea, but I would make a different endpoint for this kind of request, or at least not put it in the "func" interface, where it might conflict with a regular function named "tether".